src/components/ImageCard/_hotspot.scss

```diff
@@ -133,10 +133,6 @@ $selected-border: solid $selected-border-width $interactive-04;
   .#{$iot-prefix}--hotspot-content-title-wrapper--editable {
     $negative-margin-to-overide-parent-padding: calc(-1 * (#{$padding}));
     margin: $negative-margin-to-overide-parent-padding;
-    width: calc(
```

Comment:
Isn't this the same declaration found just below on line 139 (after change)? Would they both need to be chagned?

Reply:
Nope - that one below actually uses the correct interpolation syntax for `#{$selected-border-width}`